### PR TITLE
Stringify CQL Output that will be inserted into a CommunicationRequest.payload

### PR DESCRIPTION
**Pull request by mickohanlon23**
Addresses Issue #5

### CODE REVIEW

**Comment by dwwinters** [comment]
With the two most recent commits on this branch, I'm happy with the implementation.  @mickohanlon23 you can merge whenever you like.

**Review by dwwinters — APPROVED** [review]

File: src/main.js [hunk 1]
```diff
@@ -563,14 +565,15 @@ function formatErrorMessage(errorOutput) {
         const value = await evaluateExpression(dV.expression.expression);
         return {
           path: dV.path,
-          evaluated: value
+          evaluated: typeof value === 'string' ? value : JSON.stringify(value)
```
Comment (mickohanlon23) [inline src/main.js]: @dwwinters Should we remove the changes here now that we are stringifying the CQL output on line 575 below?

File: test/main.test.js [hunk 2]
```diff
@@ -447,6 +447,46 @@ describe('CQL expression tests', async function() {
...
+
+    otherResources.should.deep.equal([
+      {
+        id: '60',
+        subject: { reference: 'Patient/1', display: '' },
+        resourceType: 'CommunicationRequest',
+        status: 'option',
+        payload: [
+          {
+            content: "{\"coding\":[{\"system\":\"http://snomed.info/sct\",\"code\":\"10828004\",\"display\":\"Positive\"}],\"text\":\"I'm something\"}"
```
Comment (mickohanlon23) [inline test/main.test.js]: @dwwinters Is there a way that we can make the element here `contentString` in the actual FHIR Resource?

File: src/main.js (hunk 1 above)
Comment (dwwinters) [reply src/main.js]: Darn.  Yes.

File: test/main.test.js (hunk 2 above)
Comment (dwwinters) [reply test/main.test.js]: That's a tough one.  To do this we need to differentiate between elements that are strings and choice elements which could be strings.  I'll create another issue for that.

Comment (dwwinters) [reply test/main.test.js]: @mickohanlon23 I ended up coming up with a fix for this; that's included in the latest commits.

**Pull request closed without merging**